docs/source/support/must-gather.md

```diff
@@ -1,12 +1,16 @@
 # Gathering data with must-gather
 
 `must-gather` is an embedded tool in Scylla Operator that helps collecting all the necessary info when something goes wrong.
-
 The tool talks to the Kubernetes API, retrieves a predefined set of resources and saves them into a folder in your current directory.
-By default, all collected Secrets are censored to avoid sending sensitive data.
-That said, you can always review the archive before you attach it to an issue or your support request.
 
-Given it needs to talk to the Kubernetes API, at the very least, you need to supply the `--kubeconfig` flag with a path to the kubeconfig file for your Kubernetes cluster, or set the `KUBECONFIG` environment variable.
+:::{note}
+By default, sensitive resources like `Secrets` or `bitnami.com.SealedSecrets` are omitted from the collection.
+If you ever need to include them (ScyllaDB support will explicitly ask for that if a need arises), please refer to the [include sensitive resources](#including-sensitive-resources) section below.
```

Comment:
The reference to ScyllaDB Support is problematic because operator per se is OSS. Suggested rewording.

```suggestion
If you come across a clear need to include them (not recommended unless the default settings are deemed insufficient for investigation), please refer to the [include sensitive resources](#including-sensitive-resources) section below.
```